app/controllers/navigation/Navigator.scala

```diff
@@ -54,16 +54,19 @@ object Navigator {
 
   val standard: PartialFunction[DeclarationFieldCompanion, Mode => Call] = {
     case BorderTransport => controllers.declaration.routes.DepartureTransportController.displayPage
+    case PreviousDocumentsData => controllers.declaration.routes.NatureOfTransactionController.displayPage
```

Comment:
If we're using page names here, my propose is to use `PreviousDocuments` without Data

Reply:
This has been addressed.
We do not have `PreviousDocuments` we have `Document`, i.e., `PreviousDocumentsData` is a sequence of `Document`